specification/trace/semantic_conventions/cloudevents.md

```diff
@@ -0,0 +1,51 @@
+# CloudEvents
+
+This document defines how to apply semantic conventions when instrumenting code that uses [CloudEvents](https://cloudevents.io/).
+These semantic conventions are general and not tied to any protocol used to send/receive the events.
+
+**Status**: [Experimental](../../document-status.md)
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Span name](#span-name)
+- [Span kind](#span-kind)
+- [Attributes](#attributes)
+
+<!-- tocstop -->
+
+## Span name
+
+The span name SHOULD be set to the `cloudevents.` prefix, followed by the event type in the following format:
```

Comment:
We typically don't use a "type" prefix. So using just the event type name sounds more consistent with our other semantic conventions.

Reply:
That's true but I wonder if it wouldn't make more sense to have the "type" in the span name so once can tell right from looking at the span what this is about without having to find the most specific span attributes being present. It would also make clear according to which semconv the span name is composed. Either way, nothing that would fit into this PR.
I think you could leave the cloudevents.* in though, we have the same thing in the HTTP semconv, for example.

Reply:
>can tell right from looking at the span what this is about without having to find the most specific span attributes being present

That was exactly my reasoning. In a myriad of spans I think having these with `cloudevents` help distinguish them. Also had the same thought that the HTTP spans also have this "prefix" more or less.

Reply:
For HTTP, yes but only as a fallback if you use just the HTTP verb (since just GET would be a unhelpful span name).
With cloud events, I think the event name will usually make a good span name, even without a cloudevent prefix. But if we only trace the emitting itself not the operation that the event is about then using a prefix would make sense (though then I would use `CloudEvent ` as prefix not `cloudevent.`)

Reply:
I modified the instructions to use `CloudEvents {event type}` as the span name.